Editor/foleys_PropertiesEditor.cpp

```diff
@@ -90,6 +90,12 @@ void PropertiesEditor::setStyle (juce::ValueTree styleToEdit)
     style.addListener (this);
 }
 
+void PropertiesEditor::setNodeToEdit (juce::ValueTree node)
+{
+    propertiesModel.setNodeToEdit (node);
+    propertiesList.updateContent();
+}
+
```

Comment:
I think if we do that, we need to change the PropertiesEditor a bit, since there are properties, that apply only to nodes and which must be present...